tests/integration/cloud/rest_contracts/conftest.py

```diff
@@ -166,7 +161,7 @@ def _run_pact_test(
 
         request: dict[str, str | PactBody] = {
             "method": contract_interaction.method,
-            "path": str(path),
+            "path": str(contract_interaction.request_path),
```

Comment:
`non-blocking`
Do you need to convert to a string here? It should be converted to a string inside the f-string on line `186` automatically.
```suggestion
            "path": contract_interaction.request_path,
```

Reply:
Yes, I'm unpacking this dict on line 182